src/Init/LeanInit.lean

```diff
@@ -223,12 +223,12 @@ def isOfKind : Syntax → SyntaxNodeKind → Bool
 def getArg (stx : Syntax) (i : Nat) : Syntax :=
 match stx with
 | Syntax.node _ args => args.get! i
-| _                  => arbitrary _
+| _                  => panic! "Syntax.getArg: not a node"
 
 def getArgs (stx : Syntax) : Array Syntax :=
 match stx with
 | Syntax.node _ args => args
-| _                  => #[]
+| _                  => panic! "Syntax.getArgs: not a node"
```

Comment:
Do you agree with this change? It definitely makes debugging much easier.

Reply:
The names could be changed to reflect the possibility of panics, thought this was already true for `getArg`.

Reply:
These changes are problematic. Users may create buggy macros that produce invalid `Syntax` objects for the builtin elaboration functions. We use `getArg` and `getArgs` all over the place in the elaborator.